src/commands/api/deployWorkspaceProjectApi.ts

```diff
@@ -38,12 +48,14 @@ export async function deployWorkspaceProjectApi(deployWorkspaceProjectOptions: a
             rootFolder,
             srcPath: srcPath ? Uri.file(srcPath).fsPath : undefined,
             dockerfilePath: dockerfilePath ? Uri.file(dockerfilePath).fsPath : undefined,
+            newRegistryCredentialType: RegistryCredentialType.DockerLogin,
```

Comment:
We need discussions about whether we want to default to managed identity here in the future.  For now, I'm keeping it was `Docker Login` to preserve existing behavior.